src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -1508,24 +1521,18 @@ namespace OutdoorAirUnit {
 				Node( InletNode ).MassFlowRate = OutAirUnit( OAUnitNum ).ExtAirMassFlow;
 			}
 
-			//Air mass balance check (removed because exhaust and supply can be imbalanced
-			//    IF ((Node(InletNode)%MassFlowRate > Node(OutsideAirNode)%MassFlowRate) &
-			//             .OR.(Node(InletNode)%MassFlowRate < Node(OutsideAirNode)%MassFlowRate)) THEN
-			//      OutAirUnit(OAUnitNum)%UnBalancedErrCount = OutAirUnit(OAUnitNum)%UnBalancedErrCount + 1
-			//      IF (OutAirUnit(OAUnitNum)%UnBalancedErrCount .EQ. 1) THEN
-			//        CALL ShowWarningError('Air mass flow between zone supply and exhaust is not balanced')
-			//        CALL ShowContinueError('Occurs in ' // 'ZoneHVAC:OutdoorAirUnit' // ' Object=' &
-			//                               //TRIM(OutAirUnit(OAUnitNum)%Name))
-			//        CALL ShowContinueError('Air mass balance is required by other outdoor air units,'// &
-			//                                  'ZoneMixing, ZoneCrossMixing, or other air flow control inputs.')
-			//!
-			//!  CALL ShowContinueErrorTimeStamp('Air volume flow rate ratio = '//TRIM(RoundSigDigits(HXAirVolFlowRatio,3))//'.')
-			//!ELSE
-			//! CALL ShowRecurringWarningErrorAtEnd(TRIM(OutAirUnit(OAUnitNum)%Name)//&
-			//! ':  Air mass balance is required by other outdoor air units, ZoneMixing, ZoneCrossMixing, or other air flow control inputs.'&
-			//!   , OutAirUnit(OAUnitNum)%UnBalancedErrIndex)
-			//      END IF
-			//    END IF
+			// Air mass balance check
+			if ( std::abs( OutAirUnit( OAUnitNum ).ExtAirMassFlow - OutAirUnit( OAUnitNum ).OutAirMassFlow ) > 0.001 ) {
+				OutAirUnit( OAUnitNum ).UnBalancedErrCount = OutAirUnit( OAUnitNum ).UnBalancedErrCount + 1;
+				if ( OutAirUnit( OAUnitNum ).UnBalancedErrCount == 1 ) {
+					ShowWarningError( "Air mass flow between zone supply and exhaust is not balanced" );
```

Comment:
@lgu1234 Does this warning now happen for any unbalanced on a single OA unit?  Even if another unit balances everything?  Even if there is no airloop serving the same zone?  If so, then the docs should be changed to say this unit must be balanced.  Or perhaps better yet, the OA unit should be placing the correct flow on an exhaust node whether there is an exhaust fan or not.  But the exhaust node "Air Inlet Node Name" which is munged to "AirInlet Node Name" in the IDD is not a required field.  And this warning should probably not be triggered if zone mass conservation is active?  If we go ahead and add these changes, we still need  followup with a better fix after v8.8 release.